safe/gui/tools/shake_grid/shakemap_converter_dialog.py

```diff
@@ -111,6 +111,25 @@ def on_input_path_textChanged(self):
             self.get_output_from_input()
         self.update_warning()
 
+    # noinspection PyPep8Naming
+    def on_input_place_textChanged(self):
+        """Action when input place layer name is changed."""
+        if os.path.exists(self.input_place.text()):
+            try:
+                self.place_layer = QgsVectorLayer(
+                    self.input_place.text(),
+                    'Nearby Cities',
```

Comment:
Use the `tr` function if we keep this layer in the map canvas for the user.

Reply:
sorry I forgot to use the `tr`, will include it in my next commit